dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/dialogEditBindDomain.xml

```diff
@@ -18,63 +29,62 @@
         <help>Set the IP address of master server when using slave mode.</help>
     </field>
     <field>
-        <id>domain.domainname</id>
-        <label>Domain Name</label>
-        <type>text</type>
-        <help>Set the name for this ACL.</help>
+        <label>Master Zone</label>
+        <type>header</type>
+        <style>zone_type zone_type_master</style>
     </field>
     <field>
         <id>domain.allowtransfer</id>
         <label>Allow Transfer</label>
         <type>dropdown</type>
-        <help>Define an ACL where you allow which server can retrieve your zone.</help>
+        <help>Define an ACL where you allow which server can retrieve this zone.</help>
     </field>
     <field>
         <id>domain.allowquery</id>
         <label>Allow Query</label>
         <type>dropdown</type>
-        <help>Define an ACL where you allow which client are allowed to query this domain.</help>
+        <help>Define an ACL where you allow which client are allowed to query this zone.</help>
     </field>
     <field>
         <id>domain.ttl</id>
         <label>TTL</label>
         <type>text</type>
-        <help>Set the general TTL for this domain.</help>
+        <help>Set the general Time To Live for this zone.</help>
     </field>
     <field>
         <id>domain.refresh</id>
         <label>Refresh Time</label>
         <type>text</type>
-        <help>Set the time in seconds.</help>
+        <help>Set the time in seconds after which secondary name servers should refresh the zone information.</help>
```

Comment:
Can you check if secondary is true? Maybe because you only use it for secondary name server, but these values are also valid for caching nameservers.

Reply:
Caching nameservers are also secondary nameservers ;)
If you like I could rephrase it to "...secondary name servers and caching name servers should refresh....".

Reply:
Can we agree to just use name servers .. so it includes caching, resolver, slaves etc.? 
Already did a version bump? Sorry for the delay, was a tough week and forgot about you

Reply:
@mimugmail Fixed in my most recent commit, it now uses just "name servers". :)